packages/wonder-blocks-core/src/components/__tests__/unique-id-provider.test.js

```diff
@@ -171,11 +194,15 @@ describe("UniqueIDProvider", () => {
             );
 
             // Act
-            mount(nodes);
+            render(nodes);
 
             // Assert
-            expect(foo).toHaveBeenCalled();
-            expect(foo.mock.calls[0][0]).toBeTruthy();
+            waitFor(() => {
+                expect(foo).toHaveBeenCalled();
+            });
+            waitFor(() => {
+                expect(foo.mock.calls[0][0]).toBeTruthy();
+            });
```

Comment:
**question**: Shouldn't this be a single `waitFor` containing all the expectations? Also, doesn't `waitFor` need to be `await`'d?